tests/helpers.py

```diff
@@ -62,3 +73,85 @@ def dict_to_item(raw,convert_root=True):
         item =  {'N': str(raw)}
 
     return item if convert_root else item['M']
+
+def mock_state_handler_db_context():
```

Comment:
This should be renamed to `mock_execution_results_table_entry` since its a mock of an entry in the execution results table